src/pages/projects/sistent/components/list/code.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
```

Comment:
This file is not required, as pages will be dynamically created. so you don't require to create any pages under this dirc `src/pages/projects/sistent/components/list/`